### PR TITLE
[Func/Arith] Supplemental information for Trigonometric functions

### PR DESCRIPTION
**Pull request by sanjibansg**
This PR adds the supplemental information for Trigonometric functions.

### CODE REVIEW

File: supplemental/arithmetic/cosh.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Cosh
+
+## Options
+
+### Rounding
+
+Hyperbolic cosine of an input can yield a result that is not exactly
```
Comment (EpsilonPrime) [inline supplemental/arithmetic/cosh.md:7]: Here you don't capitalize cosine (I prefer this form) and for all the others you did.  Could we capitalize the names consistently in one form or the other?

**Review by EpsilonPrime — APPROVED** [review]
Looks like some of the postgres checks are failing here.

File: cases/arithmetic/acosh.yaml [hunk 2]
```diff
@@ -1,14 +1,5 @@
...
 cases:
-  - group:
-      id: basic
-      description: Basic examples without any special cases
-    args:
-      - value: 0.01
-        type: fp32
-    result:
-      value: null
-      type: fp32
```
Comment (richtia) [inline cases/arithmetic/acosh.yaml:11]: why is this removed?

File: supplemental/arithmetic/cosh.md (hunk 1 above)
Comment (sanjibansg) [reply supplemental/arithmetic/cosh.md:7]: made the change, thanks.

File: cases/arithmetic/acosh.yaml (hunk 2 above)
Comment (sanjibansg) [reply cases/arithmetic/acosh.yaml:11]: Domain of `arcosh` function is [1,Infinity), so I added this case in the on_domain_error group.
